webpage/language/nl.js

```diff
@@ -0,0 +1,11 @@
+let TRANSLATION_NL = {
+	"menu": {
+		"title": "Woord Klok",
+		"language": {
+			"label": "Taal",
+			"de": "🇩🇪 Duits",
+			"en": "🇺🇸 Engels",
+			"nl": "🇳🇱 Nederlands"
```

Comment:
This is just a preconfigured but incomplete Dutch translation file, so @pienmatla can add a Dutch translation easily once this PR is merged. The default fallback language for every missing translation is English, as configured in `i18n.js`.